docs/developing/config.md

```diff
@@ -117,6 +117,7 @@ Below are the config options that Ionic uses.
 | `backButtonDefaultHref`  | `string`                                                                          | Overrides the default value for the `defaultHref` property in all `<ion-back-button>` components.        |
 | `backButtonIcon`         | `string`                                                                          | Overrides the default icon in all `<ion-back-button>` components.                                        |
 | `backButtonText`         | `string`                                                                          | Overrides the default text in all `<ion-back-button>` components.                                        |
+| `enableHTMLContent`      | `boolean`                                                                         | Relevant Components: `ion-alert`, `ion-infinite-scroll-content`, `ion-loading`, `ion-refresher-content`, `ion-toast`. If `false`, all `innerHTML` usage will be disabled in Ionic, and custom HTML will not be usable in the relevant components. `innerHTML` usage is enabled by default. |
```

Comment:
"All `innerHTML` usage will be disabled in Ionic" makes it sound like devs can't use `innerHTML` in their own code either, which I don't believe is true? Making this more specific (closer to what you wrote in the security docs) might be better.